indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamPartitions.java

```diff
@@ -31,14 +31,13 @@
 /**
  * class that encapsulates a partitionIdToSequenceNumberMap of partitionId -> sequenceNumber.
  * To be backward compatible with both Kafka and Kinesis datasource metadata when
- * deserializing json. Redundant constructor fields stream, topic and
+ * serializing and deserializing json, redundant constructor fields stream, topic and
```

Comment:
nit: " redundant constructor fields stream, topic, ~and~ partitionSequenceNumberMap and partitionOffsetMap"

Reply:
Thanks, fixed.